### PR TITLE
Freebsd support

### PR DESCRIPTION
**Pull request by goldhoorn**

### CODE REVIEW

File: lib/autobuild/import/archive.rb [hunk 1]
```diff
@@ -183,6 +183,9 @@ def update_cache(package)
                         if retries = self.retries
                             additional_options << "--tries" << retries
                         end
+			if Autobuild.bsd?
+                            additional_options << "--no-check-certificate"
```
Comment (doudou) [inline lib/autobuild/import/archive.rb]: ???? More info on that ? I definitely don't want to sneak something that critical like this !

Comment (goldhoorn) [inline lib/autobuild/import/archive.rb]: On 23.03.2015 15:53, Sylvain Joyeux wrote:
>
> In lib/autobuild/import/archive.rb 
> <https://github.com/rock-core/autobuild/pull/19#discussion_r26943188>:
>
> > @@ -183,6 +183,9 @@ def update_cache(package)
> >                          if retries = self.retries
> >                              additional_options << "--tries" << retries
> >                          end
> > +			if Autobuild.bsd?
> > +                            additional_options << "--no-check-certificate"
>
> ???? More info on that ? I definitely don't want to sneak something 
> that critical like this !
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/rock-core/autobuild/pull/19/files#r26943188>.
>
It seems BSD has less certificates availble, Even with correct 
time-settings i got the error while downloading sisl



-- 
  Dipl.-Inf. Matthias Goldhoorn
  Space and Underwater Robotic

  Universität Bremen
  FB 3 - Mathematik und Informatik
  AG Robotik
  Robert-Hooke-Straße 1
  28359 Bremen, Germany
  
  Zentrale: +49 421 178 45-6611
  
  Besuchsadresse der Nebengeschäftstelle:
  Robert-Hooke-Straße 5
  28359 Bremen, Germany
  
  Tel.:    +49 421 178 45-4193
  Empfang: +49 421 178 45-6600
  Fax:     +49 421 178 45-4150
  E-Mail:  matthias.goldhoorn@informatik.uni-bremen.de

  Weitere Informationen: http://www.informatik.uni-bremen.de/robotik

Comment (doudou) [inline lib/autobuild/import/archive.rb]: can you try with another tool, like curl ?

Comment (goldhoorn) [inline lib/autobuild/import/archive.rb]: I updated the PR and changed to CURL for BSD like systems (including Macos -> Darwin is a BSD, we workarounded in the past)

For the log: Please find attached the output for wget:
```
[dev@freebsd-test /tmp]$ wget rock-robotics.org/master/autoproj_bootstrap
--2015-03-24 08:14:25--  http://rock-robotics.org/master/autoproj_bootstrap
Resolving rock-robotics.org (rock-robotics.org)... 46.252.18.41, 2a00:1158:0:300:abe6::1
Connecting to rock-robotics.org (rock-robotics.org)|46.252.18.41|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://raw.githubusercontent.com/rock-core/autoproj/stable/bin/autoproj_bootstrap [following]
--2015-03-24 08:14:25--  https://raw.githubusercontent.com/rock-core/autoproj/stable/bin/autoproj_bootstrap
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.31.17.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.31.17.133|:443... connected.
ERROR: cannot verify raw.githubusercontent.com's certificate, issued by 'CN=DigiCert SHA2 High Assurance Server CA,OU=www.digicert.com,O=DigiCert Inc,C=US':
  Unable to locally verify the issuer's authority.
To connect to raw.githubusercontent.com insecurely, use `--no-check-certificate'.
```

**Comment by goldhoorn** [comment]
*updated*

**Comment by doudou** [comment]
So ... that means that you don't have the problem with curl ?

**Comment by goldhoorn** [comment]
On 24.03.2015 13:54, Sylvain Joyeux wrote:
>
> So ... that means that you don't have the problem with curl ?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/rock-core/autobuild/pull/19#issuecomment-85484540>.
>
Correct ;)

-- 
  Dipl.-Inf. Matthias Goldhoorn
  Space and Underwater Robotic

  Universität Bremen
  FB 3 - Mathematik und Informatik
  AG Robotik
  Robert-Hooke-Straße 1
  28359 Bremen, Germany
  
  Zentrale: +49 421 178 45-6611
  
  Besuchsadresse der Nebengeschäftstelle:
  Robert-Hooke-Straße 5
  28359 Bremen, Germany
  
  Tel.:    +49 421 178 45-4193
  Empfang: +49 421 178 45-6600
  Fax:     +49 421 178 45-4150
  E-Mail:  matthias.goldhoorn@informatik.uni-bremen.de

  Weitere Informationen: http://www.informatik.uni-bremen.de/robotik

**Pull request merged**